1-js/02-first-steps/13-switch/article.md

```diff
@@ -109,6 +109,32 @@ switch (+a) {
     alert('no-no, see the code above, it executes');
 }
 ```
+
```

Comment:
Well IMHO when you say that the `case` clause can have an expression and not necessarily a constant, then I believe it helps to explain which of these expressions are actually evaluated.

It's a little bit like when you explain that when evaluating a logical AND (&&), if the first expression is false then the second one does not get evaluated at all.

This being said, this is not crucial and I would totally understand if you think the addition is more confusing than helpful.